### PR TITLE
Added 'tag_on_failure' attribute

### PR DESCRIPTION
**Pull request by hartfordfive**

### CODE REVIEW

File: spec/filters/kv_spec.rb [hunk 1]
```diff
@@ -107,6 +107,21 @@
...
+      context "without splitter" do
+        let(:tag_1) { "_failure_tag_1" }
+        let(:tag_2) { "_failure_tag_2" }
+        let(:message) { "random text foo bar baz" }
+        let(:options) { {"tag_on_failure" => ["#{tag_1}", "#{tag_2}"]} }
+
+        it "should short circuit" do
+          subject.filter(event)
+          expect(event['tags']).to include("_failure_tag_1")
+          expect(event['tags']).to include("_failure_tag_2")
```
Comment (guyboertje) [inline spec/filters/kv_spec.rb]: given that you defined `tag_1` and `tag_2` in let blocks, you should consider using them in lines 120, 121

**Review by guyboertje — COMMENTED** [review]
One comment otherwise LGTM - needs a rebase + rework though

**Comment by guyboertje** [comment]
@hartfordfive 
Deep apologies for the delay.

Are you still able to proceed with this? If so please rebase and consider my comment.

File: spec/filters/kv_spec.rb (hunk 1 above)
Comment (hartfordfive) [reply spec/filters/kv_spec.rb]: Thank you for pointing that out, definitely an oversight on my part.  I'll try to rebase my branch and push these changes within the next day.

**Comment by colinsurprenant** [comment]
Thanks for your contribution @hartfordfive. 
This feature was added to the filter. Closing.

File: spec/filters/kv_spec.rb (hunk 1 above)
Comment (benmvp) [reply spec/filters/kv_spec.rb:121]: Acknowledged. The Phase 1C cutover removes the old control-loop ownership and routes the lifecycle through the Runtime and Harness boundary.

**Pull request closed without merging**